### PR TITLE
improved introduction in index.md

### PR DESCRIPTION
**Pull request by wonk1132**
I think giving a little context about how Cats relates to category theory and Haskell would be good.

Added a little blurb about what category theory is. Added a paragraph about learning more about category theory.

### CODE REVIEW

**Comment by peterneyens** [comment]
I think this would make the introduction a lot less beginner friendly and too complex for a project introduction in general.

I'm fine with adding Bartosz Milewski's website and YouTube channel to the resources for learners. Maybe you could add something along the lines of your intro additions to the resources instead (before the links) ?

**Comment by wonk1132** [comment]
sounds good. I moved the text as you suggested.

**Comment by easel** [comment]
I like these additions to the docs, @wonk1132 any change you can bring this PR up to date?

File: docs/src/main/tut/resources_for_learners.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ title:  "Resources for Learners"
 section: "resources_for_learners"
 position: 30
 ---
+
+The name *Cats* is a playful shortening of the word *category*. This term derives from
+*Category theory*, a branch of abstract mathematics with far reaching implications
```
Comment (ceedubs) [inline docs/src/main/tut/resources_for_learners.md]: Minor: "Category" probably doesn't need to be capitalized here.

File: docs/src/main/tut/resources_for_learners.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ title:  "Resources for Learners"
...
+in Computer Science that
+gained popularity in [Haskell](https://www.haskell.org/). the Cats library implements
+Category theory concepts in Scala, owing a debit to Haskell's pioneering efforts to codify
```
Comment (ceedubs) [inline docs/src/main/tut/resources_for_learners.md]: Minor typo: debit -> debt

File: docs/src/main/tut/resources_for_learners.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ title:  "Resources for Learners"
...
+Category theory concepts in a language. Cats has many similiarities to Haskell's codification
+of Category theory into a computer language, but with some improvements along with implementation
```
Comment (ceedubs) [inline docs/src/main/tut/resources_for_learners.md]: Minor: the word "improvements" makes me a bit uneasy here. What about something like "but with some adjustments to better fit Scala and the JVM"?

**Comment by kailuowang** [comment]
Closing stale PRs. Feel free to reopen if there is interest to revive the effort.

**Comment by kailuowang** [comment]
Closing stale PRs. Feel free to reopen if there is interest to revive the effort.

**Pull request closed without merging**